package.json

```diff
@@ -3,13 +3,10 @@
   "displayName": "Prisma",
   "description": "Adds syntax highlighting, formatting, auto-completion, jump-to-definition and linting for .prisma files.",
   "version": "0.0.31",
-  "prisma": {
-    "version": "76857c35ba1e1764dd5473656ecbbb2f739e1822"
-  },
```

Comment:
Did this move somewhere else? It's important in defining which binary versions the Extension should download and use.

Reply:
yes, that moved to the server package.json